astrodb_utils/loaders.py

```diff
@@ -0,0 +1,187 @@
+import logging
+import os
+import tomllib
+from dataclasses import dataclass
+
+from astrodbkit.astrodb import Database, create_database
+
+from astrodb_utils.utils import AstroDBError
+
+__all__ = [
+    "build_db_from_json",
+    "read_db_from_file",
+    "DatabaseSettings",
+]
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class DatabaseSettings:
+    settings_file: str = "database.toml"
+    base_path: str = "."
+    db_name: str = None
+    felis_path: str = None
+    data_path: str = None
+    lookup_tables: list = None
+
+    def __post_init__(self):
+        # Use a base_path if provided, otherwise default to the current directory
+        if self.base_path is None:
+            self.base_path = "."
```

Comment:
What's the point of setting a default? Isn't this redundant?

Reply:
The calling function sets it as None, but I can make some changes to simplify that.